iocore/net/SSLNetVConnection.cc

```diff
@@ -1364,6 +1365,7 @@ SSLNetVConnection::sslServerHandShakeEvent(int &err)
 
 #if TS_USE_TLS_ASYNC
   case SSL_ERROR_WANT_ASYNC:
+    SSL_INCREMENT_DYN_STAT(ssl_error_async);
```

Comment:
Adding this metric on another commit (PR) would make future debugging easy, because you'd lose the metric if you had to revert this entire change.